src/Models/ProjectModel.php

```diff
@@ -27,13 +27,28 @@ public function getProjects(): array
         return $query->fetchAll();
     }
 
```

Comment:
Annoyingly I can't put a comment on the right line. Suggest renaming getProjects to just be getAll()